Directory.Build.props

```diff
@@ -21,8 +21,8 @@
 		<NetCurrent>net9.0</NetCurrent>
 		<NetPrevious>net8.0</NetPrevious>
 
-		<NetCurrentNetCoreMobile>$(NetCurrent)-android;$(NetCurrent)-ios17.2;$(NetCurrent)-maccatalyst17.2;$(NetCurrent)-macos14.2</NetCurrentNetCoreMobile>
-		<NetPreviousNetCoreMobile>$(NetPrevious)-android;$(NetPrevious)-ios17.0;$(NetPrevious)-maccatalyst17.0;$(NetPrevious)-macos14.0</NetPreviousNetCoreMobile>
+		<NetCurrentNetCoreMobile>$(NetCurrent)-android;$(NetCurrent)-ios17.5;$(NetCurrent)-maccatalyst17.5;$(NetCurrent)-macos14.2</NetCurrentNetCoreMobile>
+		<NetPreviousNetCoreMobile>$(NetPrevious)-android;$(NetPrevious)-ios17.5;$(NetPrevious)-maccatalyst17.5;$(NetPrevious)-macos14.0</NetPreviousNetCoreMobile>
```

Comment:
https://github.com/xamarin/xamarin-macios/issues/21093#issuecomment-2312468294

Reply:
Ah ok, will not need this, as long as the version is replaced in Samples app with override, will adjust things